testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/mdb/dynamic/DynamicMessageListenerTestCase.java

```diff
@@ -88,7 +88,11 @@ public static Archive createDeplyoment() {
                 new RuntimePermission("defineClassInPackage." + MyMdb.class.getPackage().getName()),
                 new RuntimePermission("getClassLoader"),
                 // TelnetServer binds socket and accepts connections
-                new SocketPermission(Utils.getDefaultHost(true), "accept,listen,resolve")),
+                new SocketPermission(Utils.getDefaultHost(true), "accept,listen"),
+                /**
+                 * It have to be here. {@link java.lang.SecurityManager#checkListen(int)}
+                 */
+                new SocketPermission("localhost", "listen")),
```

Comment:
Should 91-95 just be:

new SocketPermission("*", "accept,listen"),


TelnetServer is opening a socket that listens on the wildcard address:

this.serverSocket = new ServerSocket(port);

Using "*" should deal with the fact in a more robust manner plus get around the seemingly bizarre thing in SecurityManager.checkListen, since "*" will match localhost.